packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -4868,5 +4868,6 @@ void main() {
       find.byType(MaterialApp),
       matchesGoldenFile('selectable_text_golden.TextSelectionStyle.2.png'),
     );
-  });
+    // This golden file test is marked negative on web
+  }, skip: isBrowser); // https://github.com/flutter/flutter/issues/93719
```

Comment:
Thanks for skipping this! Could you please link to this issue instead: https://github.com/flutter/flutter/issues/93723

Reply:
Gotcha, thanks!